### PR TITLE
Added crypto_hash_sha256 with testcase and documentation

### PR DESCRIPTION
**Pull request by ne0h**

### CODE REVIEW

**Comment by ne0h** [comment]
I am sorry that I produced such a chaos...

File: pysodium/__init__.py [hunk 1]
```diff
@@ -423,3 +423,14 @@ def crypto_sign_sk_to_pk(sk):
     res = ctypes.create_string_buffer(crypto_sign_ed25519_PUBLICKEYBYTES)
     __check(sodium.crypto_sign_ed25519_sk_to_pk(ctypes.byref(res), sk))
     return res.raw
+
+# int crypto_hash_sha256(unsigned char *out, const unsigned char *in,
+#                       unsigned long long inlen);
+def crypto_hash_sha256(message):
+    if message is None:
+        raise ValueError("invalid parameters")
+    # TODO: use crypto_hash_sha256_BYTES from libsodium here.
```
Comment (stef) [inline pysodium/__init__.py]: could you please fix this todo?

**Comment by stef** [comment]
very nice PR, would you please fix this one const todo item? then i merge, and i guess i also release a new version on pypi

**Comment by ne0h** [comment]
That did not work on OSX yesterday. Now I tested again and found a typo :D

**Comment by stef** [comment]
perfect! thx

**Pull request merged**